cmake/std/atdm/ats1/environment.sh

```diff
@@ -83,50 +68,15 @@ export ATDM_CONFIG_SBATCH_EXTRA_ARGS="$ATDM_CONFIG_SBATCH_EXTRA_ARGS --exclude=n
 export OMP_NUM_THREADS=2
 
 # Common modules and paths
-module load cmake/3.14.6
-module load git/2.19.1
+module load sparc-cmake/3.18.1
+module load git/2.21.0
```

Comment:
And let's remove this `module load git/2.21.0` and just use what the `sparc-dev` module loads which is:

```
$ ssh mutrino

$ which git
/usr/bin/git

$ git --version
git version 2.26.2

$ module load sparc-dev/intel-19.0.4_mpich-7.7.15_hsw

$ which git
/usr/projects/sparc/tools/ats1/git-2.19.1/intel-18.0.2/bin/git

$ git --version
git version 2.19.1
```

Let's not muck with the env loaded by the `sparc-dev` module unless we absolutely have to.  That way, there is less work to upgrade as time goes on.